packages/kbn-search-connectors/components/sync_jobs/documents_panel.tsx

```diff
@@ -30,7 +30,7 @@ export const SyncJobDocumentsPanel: React.FC<SyncJobDocumentsPanelProps> = (sync
           content={
             <FormattedMessage
               id="searchConnectors.index.syncJobs.documents.upserted.tooltip"
-              defaultMessage="The number of {index} operations the connector sent to the Elasticsearch _bulk API during this sync. This includes net-new documents and updates to existing documents. This does not account for duplicate _ids, or any documents dropped by an ingest processor"
+              defaultMessage="Number of {index} operations the connector sent to the Elasticsearch _bulk API, including updates to existing documents. Note that the number of documents upserted and the number of documents in the index may differ."
```

Comment:
This loses the _why_ they might differ. And I get that this maybe doesn't belong in a tooltip. Do you think there's a good place in the docs where we could add the "why" if we can't put it here? We're trying to reduce how many support cases are filed when these numbers don't line up.

Reply:
Flipping this to ready to review, expecting CI failures because wasn't able to build locally originally.

Reply:
> expecting CI failures because wasn't able to build locally originally

I pushed a fix to one test I saw was going to fail. I don't expect other CI issues, at least not ones related to these changes.

Reply:
https://github.com/elastic/kibana/pull/181308/commits/b14151748e395431aa7fdf93066c863274b621e7 deletes a line that got added (I think by mistake)